src/main/java/edu/umich/its/cpm/Utils.java

```diff
@@ -314,7 +315,7 @@ public static boolean isCurrentUserCPMAdmin(HttpServletRequest request,
 	 * get CoSign user
 	 */
 	private static String getRemoteUser(HttpServletRequest request) {
-		return request.getRemoteUser();
+		return "zqian";//request.getRemoteUser();
```

Comment:
How about making this a property?  Return the test user if the property is set otherwise return the remote user.